demisto_sdk/commands/validate/validators/GR_validators/GR105_duplicate_content_id.py

```diff
@@ -97,17 +99,43 @@ def obtain_invalid_content_items_using_graph(
                 for content_item in content_items
             ]
         )
-        return [
-            ValidationResult(
-                validator=self,
-                message=self.error_message.format(
-                    content_item.object_id,
-                    Path(duplicate.path).relative_to(CONTENT_PATH),  # type: ignore[attr-defined]
-                ),
-                content_object=content_item,  # type: ignore[arg-type]
-            )
-            for content_item, duplicates in self.graph.validate_duplicate_ids(
-                paths_of_content_items_to_validate
-            )
-            for duplicate in duplicates
-        ]
+        results = []
+
+        for content_item, duplicates in self.graph.validate_duplicate_ids(
+            paths_of_content_items_to_validate
+        ):
+            if duplicates:
+                if invalid_duplicates([*duplicates, content_item]):
+                    results.extend(
+                        [
+                            ValidationResult(
+                                validator=self,
+                                message=self.error_message.format(
+                                    content_item.object_id,
+                                    Path(duplicate.path).relative_to(CONTENT_PATH),  # type: ignore[attr-defined]
+                                ),
+                                content_object=content_item,  # type: ignore[arg-type]
+                            )
+                            for duplicate in duplicates
+                        ]
+                    )
+
+        return results
+
+
+def invalid_duplicates(items: list):
```

Comment:
please check if this is not something that we already do [here](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/content_graph/interface/neo4j/queries/validations.py#L316)

Reply:
we dont, it's part of the newly added check.